prow/config.yaml

```diff
@@ -10769,6 +10804,74 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
+- cron: "0 3,9,15,21 * * *" # every six hours, disjoint with ci-kubernetes-e2e-gci-gce-scalability-stable1
```

Comment:
Instead of listing each hour exhaustively, you can do `3-21/6` - which evaluates to the residue class `3 modulo 6`

Reply:
...in that range

Reply:
done